### PR TITLE
added unique_key to the test_docs_generate places relevant to changes…

### LINKED ISSUE #112: [CT-116] Add integration test for new list input for unique ID
This PR (https://github.com/dbt-labs/dbt-core/pull/4618) added the ability to pass a list for unique ID. We would like to have an integration test for BigQuery that verifies that holds true here

### PR DESCRIPTION
**Pull request by McKnight-42**
… made in core, TODO: double check that any other references to unique_key in bigquery there are two more on that file need to be changed to None value as well, and design an integration test

resolves #112 

### Description

making changes to be in line with changes to dbt-core regarding use of lists in unique_key values. [#4618](https://github.com/dbt-labs/dbt-core/pull/4618)

### Checklist

- [x] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [x] I have run this code in development and it appears to resolve the stated issue
- [x] This PR includes tests, or tests are not required/relevant for this PR
- [x] I have updated the `CHANGELOG.md` and added information about my change to the "dbt-bigquery next" section.

### CODE REVIEW

File: tests/integration/incremental_schema_tests/test_incremental_schema.py [hunk 1]
```diff
@@ -140,3 +140,4 @@ def test__bigquery__run_incremental_sync_all_columns(self):
     @use_profile('bigquery')
     def test__bigquery__run_incremental_fail_on_schema_change(self):
         self.run_incremental_fail_on_schema_change()
+
```
Comment (leahwicz) [inline tests/integration/incremental_schema_tests/test_incremental_schema.py:143]: Nit: If there is no real change to this file and just the newline, can we get rid of this change so it doesn't show an edit happened in this file?

Comment (McKnight-42) [reply tests/integration/incremental_schema_tests/test_incremental_schema.py:143]: I believe this is a pep8 thing, was also showing up in a lot of the pre-commit stuff i ran earlier as well. I can remove the line then possibly just have it in with all those on that pr.

**Comment by VersusFacit** [comment]
Just logging for posterity that Matt and I should be syncing up Tuesday this week to collate all test changes once we have some confirmation on what code changes need to be made to my Snowflake version of this PR. We all luck, we'll have an approval on this then!

**Comment by VersusFacit** [comment]
I'm going to approve this because we adapted my code which was thoroughly vetted by Kyle and Chenyu and I believe Leah herself! The snowflake tests were merged, so we'll merge this as well.

**Review by VersusFacit — APPROVED** [review]

**Pull request merged**